etc/cp-ksql-server/README.md

```diff
@@ -0,0 +1,41 @@
+# kafka-data transformer (KSQLDB)
```

Comment:
Please add a link to this document in README

Reply:
Also maybe create a `/docs` folder and put this file in there, so in future we can keep adding the documentation to that folder.

Reply:
Fixed!